### DISCUSSION TITLE
How update 'Source' plugin to download multiple .torrent files and save them in a single .zip file?

### DISCUSSION THREAD

**Discussion by kdantas** [Novik/ruTorrent#2386] [General]
Hi, my seedbox with rTorrent/ruTorrent was installed through the arakasi script. The seedbox is running on Ubuntu 16.04 Server Armhf_32. I know it's obsolete, but it's ok for me. The ruTorrent is v3.8 and the rTorrent is v0.9.7/0.13.7. My question is: how can I update the Source Plugin to download multiple .torrent files and save them in a single .zip file?

**Comment by stickz** [comment]
hi @kdantas, you are running a very outdated version of ruTorrent we no longer support. It will be required to upgrade, if you want to use this feature. **ruTorrent v4.0** is your best option, however it will require upgrading to **Ubuntu 20.04 LTS**. If you upgrade to **ruTorrent v3.9 or v3.10** for this feature, I can't guarantee it will work very well on **Ubuntu 16.04** because this operating system version is very old.

Your install script supports configuring **Ubuntu 20.04 LTS** with **ruTorrent v4.0**. This version of ruTorrent is very stable. It has a lot of improvements. Many issues were fixed leading up to this point. **Ubuntu 20.04 LTS** is also supported until 2030. However, if you want to use autodl-irssi, you'll have to manually upgrade the version of the ruTorrent plugin. The link to the source files are available. It's not difficult.

**Reply by kdantas** [reply]
If I upgrade to ruTorrent v3.10 I can use this feature directly, or I have to include extra codes?
